web/client/utils/GeoStoryUtils.js

```diff
@@ -174,7 +174,13 @@ export const applyDefaults = (options = {}) => merge({}, DEFAULT_MAP_OPTIONS, op
  * @return {object} options merged with defaults
  */
 export const createMapObject = (baseMap = {}, overrides = {}) => {
-    return merge({}, baseMap, overrides);
+    const baseMapCloned = { ...baseMap };
+    // use override [replace] logic instead of merge to get the updated map
+    const updatedMap = Object.keys(overrides).reduce((cumulatedMap, key) => {
+        cumulatedMap[key] = overrides[key];
+        return cumulatedMap;
+    }, baseMapCloned);
+    return updatedMap;
```

Comment:
Based on the previous comment I think we should merge all the properties but the layers key that will be overridden